domains/kazuoyuuka.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "KazuoYK",
+           "email": "d4rknewbie@gmail.com",
+           "discord": "1025056477628596255"
+        },
+    
+        "record": {
+            "TXT": "https://kazuoyk.github.io/dev/"
```

Comment:
TXT records alone do nothing, did you mean URL?

Reply:
can u delete?